package.json

```diff
@@ -91,6 +91,7 @@
   },
   "dependencies": {
     "accept-language-parser": "^1.5.0",
+    "async_hooks": "^1.0.0",
```

Comment:
The [async_hooks](https://www.npmjs.com/package/async_hooks) package exists only to prevent supply chain attacks. I don't think you need it as a dependency as it's empty anyways.

![image](https://user-images.githubusercontent.com/48728802/207083016-0dca88f0-9b93-4a06-bf3f-1ff991741e3a.png)

Reply:
@SchroederSteffen ah wow good catch! Didn't know it was part of nodejs. I'll remove it in the next patch 9.3.1. Any more suggestions? (Sorry for the somewhat clumsy maintenance). 😨

Reply:
Nothing code-wise, as I'm not familiar with `async_hooks`, sorry. 🙂 
Only the stuff mentioned in https://github.com/toonvanstrijp/nestjs-i18n/issues/424.